samples/ccr/docker-compose-metricstore.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
```

Comment:
Since we don't have any formal documentation, it'll be nice to also add some comments to the docker yml files specifying what these instances are needed for and how they're getting used.

Reply:
Sure.